airflow/providers/cncf/kubernetes/provider.yaml

```diff
@@ -89,6 +89,7 @@ operators:
   - integration-name: Kubernetes
     python-modules:
       - airflow.providers.cncf.kubernetes.operators.kubernetes_pod
+      - airflow.providers.cncf.kubernetes.operators.pod
```

Comment:
btw @potiuk triggers are part of providers but I noticed we don't specify them as part of the provider yaml. Is this expected?

Reply:
We do not have triggers in provider info - but that's a good idea to add them (we would also need to expand the CLI and provider manager to include and expose that information).

Reply:
Thanks for opening https://github.com/apache/airflow/issues/29918